Content.Server/Store/Systems/StoreSystem.Ui.cs

```diff
@@ -283,6 +283,9 @@ private void OnBuyRequest(EntityUid uid, StoreComponent component, StoreBuyListi
     /// </remarks>
     private void OnRequestWithdraw(EntityUid uid, StoreComponent component, StoreRequestWithdrawMessage msg)
     {
+        if (msg.Amount <= 0)
+            return;
```

Comment:
This would previously just throw an exception due to `ents.First()`, so it wasn't exploitable